django/db/models/query.py

```diff
@@ -678,8 +678,17 @@ def raw(self, raw_query, params=None, translations=None, using=None):
             using = self.db
         return RawQuerySet(raw_query, model=self.model, params=params, translations=translations, using=using)
 
-    def _values(self, *fields):
+    def _values(self, *fields, **expressions):
         clone = self._clone()
+
+        if expressions:
+            def alias(expression):
+                if hasattr(expression, 'resolve_expression'):
+                    return expression
+                return F(expression)
+            expressions = {name: alias(expr) for name, expr in expressions.items()}
+            clone = clone.annotate(**expressions)
```

Comment:
You end up cloning twice (the first original one, then this one) through this code path. Consider an else clause which handles the original case of just calling _clone.